src/containers/org/OrgDataSetContainer.js

```diff
@@ -116,6 +124,9 @@ const OrgDataSetContainer = ({
   const collabCount = collaborationsByDataSetId.size;
   const collabTabText = isSuccess(getCollabWithDataSetRS) ? `Collaborations (${collabCount})` : 'Collaborations';
 
+  const tags = metadata.get(TAGS, List());
+  const tagLabel = `Protections: ${tags.join(', ')}`;
```

Comment:
We'll eventually want to support any freeform text for tags, in addition to standardized ones, so I don't think we should have the `Protections: ` prefix.